data/data/coreos/rhcos.json

```diff
@@ -1,107 +1,107 @@
 {
   "stream": "rhcos-4.16",
```

Comment:
Can you make this change? Our automated script doesn't do it.

```suggestion
  "stream": "rhcos-4.17",
```

Reply:
done, I have changed it.